pkg/filewatcher/filewatcher.go

```diff
@@ -0,0 +1,67 @@
+package filewatcher
```

Comment:
+// WatchFileForChanges watches the file, fileToWatch, for changes. If the file contents have changed, the pod this
+// function is running on will be restarted.
+func WatchFileForChanges(fileToWatch string) error {
Boilerplate header is wrong for: /home/prow/go/src/sigs.k8s.io/azuredisk-csi-driver/hack/../pkg/filewatcher/filewatcher.go
make: *** [Makefile:91: verify] Error 1
+ EXIT_VALUE=2
+ set +o xtrace


you need to append  header to this file